public/video-ui/src/services/WorkflowApi.js

```diff
@@ -53,7 +62,17 @@ export default class WorkflowApi {
       needsLegal: 'NA',
       section,
       status,
-      prodOffice
+      prodOffice,
+      commentable: video.commentsEnabled,
+      commissioningDesks: video.commissioningDesks.join(),
+      lastModified: lastModifiedDate,
+      published: contentChangeDetails.published != undefined,
```

Comment:
sorry didn't catch this before, but this could be `!!publishedDate` for [simplicity](https://medium.com/@pddivine/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054)